internal/cmd/trust/trust.go

```diff
@@ -22,6 +24,8 @@ func New() *cobra.Command {
 	cmd.AddCommand(i.New(o))
 	cmd.AddCommand(addpolicykey.New(o))
 	cmd.AddCommand(removepolicykey.New(o))
+	cmd.AddCommand(addrootkey.New(o))
```

Comment:
nit: can you sort these alphabetically by the subcommand's package name? It wasn't originally the case but as this grows, let's keep them organized.

Reply:
Fixed